charts/descheduler/README.md

```diff
@@ -51,6 +51,7 @@ The following table lists the configurable parameters of the _descheduler_ chart
 | `nameOverride`                 | String to partially override `descheduler.fullname` template (will prepend the release name)                          | `""`                                 |
 | `fullnameOverride`             | String to fully override `descheduler.fullname` template                                                              | `""`                                 |
 | `schedule`                     | The cron schedule to run the _descheduler_ job on                                                                     | `"*/2 * * * *"`                      |
+| `startingDeadlineSeconds`      | If set, configure `startingDeadlineSeconds` for the _descheduler_ job                                                 | `""`                                 |
```

Comment:
Based on the actual value provided the default should be `nil`.

Reply:
Also fixed. I amended the commit to fix this.